Lib/symtable.py

```diff
@@ -311,6 +318,16 @@ def is_assigned(self):
         """Return *True* if a symbol is assigned to."""
         return bool(self.__flags & DEF_LOCAL)
 
+    def is_comp_iter(self):
+        """Return *True* if the symbol is a comprehension iteration variable.
+        """
+        return bool(self.__flags & DEF_COMP_ITER)
+
+    def is_comp_cell(self):
```

Comment:
Is it the same as `is_comp_iter`?

Reply:
Oups not at all! It's a typo, thanks for catching it!